cmd/cli/uninstall_mesh.go

```diff
@@ -93,7 +93,7 @@ func newUninstallMeshCmd(config *action.Configuration, in io.Reader, out io.Writ
 
 	f := cmd.Flags()
 	f.StringVar(&uninstall.meshName, "mesh-name", defaultMeshName, "Name of the service mesh")
-	f.BoolVarP(&uninstall.force, "force", "f", false, "Attempt to uninstall the osm control plane instance without prompting for confirmation.  If the control plane with specified mesh name does not exist, do not display a diagnostic message or modify the exit status to reflect an error.")
+	f.BoolVarP(&uninstall.assumeYes, "assume-yes", "y", false, "Assumes yes to prompts and attempt to uninstall the osm control plane instance without prompting for confirmation.")
```

Comment:
Lets only remove ` If the control plane with specified mesh name does not exist, do not display a diagnostic message or modify the exit status to reflect an error` and not change the name of the flag please

Reply:
I agree that renaming the flag here doesn't seem necessary given that would be a breaking change that needs to be documented in the release notes doc.